RileyLink/DeviceDataManager.swift

```diff
@@ -242,46 +370,45 @@ class DeviceDataManager {
 
     init() {
```

Comment:
No worries; I'm just bringing RL up to speed with Loop, so when you do the refactoring, I can ~~steal~~ reuse your awesome ideas more easily. ;)